terraform/projects/infra-security-groups/main.tf

```diff
@@ -147,6 +147,17 @@ resource "aws_security_group" "monitoring_internal_sg" {
   )}"
 }
 
+resource "aws_security_group_rule" "permit_internal_any_any" {
+  type      = "ingress"
+  from_port = 0
+  to_port   = 0
```

Comment:
please can we not be so permissive with ingress rules?  We may need to permit port 80 (and possible 8080? 9090? 9093?) but that doesn't mean we should permit the world.

Reply:
After looking at this again, I realise that this PR isn't newly introducing this behaviour; this permissive rule is already present in master.  Therefore, it's no reason to block this change.

We should still have a think about how we use our security groups but for now this PR can be merged.